extensions/wikidata/module/scripts/menu-bar-extension.js

```diff
@@ -1,134 +1,153 @@
 // Load the localization file
 var dictionary = {};
 $.ajax({
-	url : "command/core/load-language?",
-	type : "POST",
-	async : false,
-	data : {
-	  module : "wikidata",
+  url: "command/core/load-language?",
+  type: "POST",
+  async: false,
```

Comment:
It would be good if this didn't block the UI (but I know it's probably not your code).